### ISSUE TITLE
Export RGI_Emoji as a separate property

### ISSUE THREAD

**Issue by mathiasbynens** [node-unicode/node-unicode-data#59]
https://unicode.org/reports/tr51/#def_rgi_set

> ED-27. RGI emoji set — The set of all emoji (characters and sequences) covered by ED-20, ED-21, ED-22, ED-23, ED-24, and ED-25.

In other words:

```
RGI_Emoji =
    Basic_Emoji
  + Emoji_Keycap_Sequence
  + RGI_Emoji_Modifier_Sequence
  + RGI_Emoji_Flag_Sequence
  + RGI_Emoji_Tag_Sequence
  + RGI_Emoji_ZWJ_Sequence
```

All of those properties are enumerated in `emoji-sequences.txt` except for `RGI_Emoji_ZWJ_Sequence` which is in `emoji-zwj-sequences.txt`.

**Comment by mathiasbynens** [comment]
Doing this would simplify the [emoji-regex](https://github.com/mathiasbynens/emoji-regex) build script, and make it more future-proof / less likely to require changes in the future.

**Comment by mathiasbynens** [comment]
This is released in `@unicode/unicode-13.0.0@1.0.1`.
